test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -524,9 +524,9 @@ def set_vntype(self, mom, typestring='myvntype'):
         self.logger.info("Setting vntype to %s in %s on mom %s" %
                          (typestring, vntype_file, mom))
         fn = self.du.create_temp_file(body=typesting)
-        ret = self.du.run_copy(hosts=self.mom.hostname, src=fn,
-                               dest=vntype_file, sudo=True, uid='root',
-                               gid='root', mode=0644)
+        ret = self.du.run_copy(hosts=self.mom, src=fn,
```

Comment:
I think you go my previous comment wrong.. I asking to change 'self.mom.hostname' to 'mom.hostname' not 'self.mom'

Reply:
you missed this comment.

Reply:
@hirenvadalia : The caller passed mom name so it should be 'mom' not mom.hostname.